exercises/practice/high-scores/.meta/example.c

```diff
@@ -0,0 +1,52 @@
+#include "high_scores.h"
+
+#define MAX_TOP_SCORES (3)
+
+int32_t latest(const int32_t *scores, size_t scores_len)
+{
+   if (scores_len == 0) {
+      return 0;
+   }
+
+   return scores[scores_len - 1];
+}
+
+int32_t personal_best(const int32_t *scores, size_t scores_len)
+{
+   int32_t best;
+   if (scores_len == 0) {
+      return 0;
+   }
+
+   best = scores[0];
```

Comment:
```suggestion
   if (scores_len == 0) {
      return 0;
   }

   int32_t best = scores[0];
```

Since we're using C99, I'd rather keep declarations as close to use as possible.

Reply:
Done